docs/content/en/docs/tutorials/developer-journey.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Developer Journey with Buildpacks"
+linkTitle: "Developer Journey"
+weight: 100
+---
+
+Skaffold allows developers to easily transition from local development on minikube to remote development on an enterprise Kubernetes cluster managed by IT. During the transition from local to remote deployment, a security team might ask a developer to patch a library with a specific vulnerability in it. This is where Skaffold's support for buildpacks comes in handy. In this tutorial, you'll start out deploying an application locally, swap out buildpacks in the `skaffold.yaml` file to use the latest libraries, and then deploy the application to a remote Kubernetes cluster.
+
+For a guided Cloud Shell tutorial on how a developer's journey might look in adopting Skaffold, follow:
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/willisc7/skaffold-dev-journey-tutorial&cloudshell_workspace=.&cloudshell_tutorial=tutorial.md)
```

Comment:
I'd prefer if we inlined the `tutorial.md` in this page so that folks can/see run it without requiring cloud shell.

Reply:
Ok just made the changes. Let me know if I did that right.

Reply:
Resolved. Maintaining two markdown files for now. One under docs/.../tutorials and another in the tutorial repo itself.